daprdocs/content/en/reference/cli/dapr-mtls/_index.md

```diff
@@ -30,8 +30,9 @@ dapr mtls [command]
 ### Available Commands
 
 ```txt
-expiry      Checks the expiry of the root certificate
-export      Export the root CA, issuer cert and key from Kubernetes to local files
+expiry              Checks the expiry of the root certificate
+export              Export the root CA, issuer cert and key from Kubernetes to local files
```

Comment:
I notice that the wording on these CLI references is wrong since it says "for Kubernetes" in several places, when it should just say what it does and then then supported platforms tell you where this works.

So this sentence should be 
```suggestion
export              Export the root Certificate Authority (CA), issuer cert and key to local files
```

Reply:
Can you also apply this approach to the existing export and expiry commands in these files
https://docs.dapr.io/reference/cli/dapr-mtls/dapr-mtls-export/
https://docs.dapr.io/reference/cli/dapr-mtls/dapr-mtls-expiry/

For example in the export file 
1) Change "Export the root CA, issuer cert and key from Kubernetes to local files" to "Export the root Certificate Authority (CA), issuer cert and key to local files"

Reply:
Actually I have just realized that these export, expiry and renew should all have -k as an option to say that these apply to Kubernetes. No single CLI command should be tied to a single platform environment, so this is a bug we need to fix. In other words it should be this.

dapr mtls export -k -o ./certs
dapr mtls expiry -k 
dapr mtls renew -k

in the same way we have
dapr mtls -k

Reply:
> No single CLI command should be tied to a single platform environment, so this is a bug we need to fix.

There are existing commands which are like that `dapr components -k`, `dapr configurations -k` etc ... (only k8s)
Please create an issue in CLI and this can triaged for future releases. IMO it is a new feature and not a bug ... since the way we query data for self-hosted mode is completely different.